app/assets/javascripts/discourse/app/mixins/disable-sidebar.js

```diff
@@ -1,7 +1,13 @@
 import Mixin from "@ember/object/mixin";
+import deprecated from "discourse-common/lib/deprecated";
 
 export default Mixin.create({
   activate() {
+    deprecated(
```

Comment:
The mixin is only used in one internal repo, I'll follow-up soon and remove it altogether.